util/helm/helm.go

```diff
@@ -94,6 +94,8 @@ func (h *helm) Template(appName string, namespace string, opts *argoappv1.Applic
 				args = append(args, "--set", fmt.Sprintf("%s=%s", p.Name, p.Value))
 			}
 		}
+
+		args = append(args, "--set", fmt.Sprintf("argocd_revision=%s", commitSHA))
```

Comment:
Can Helm access environment variables in templates? It might be better to use that if possible.